custom bin_path fix

zwirek opened the pull request:
Fix for custom bin_path.
I was unable to set bin_path to phiremoc by using relative path.

mcustiel commented on src/Extension/Phiremock.php:
```diff
@@ -75,7 +75,7 @@ public function startProcess()
             $ip,
             $port,
-            realpath($this->config['bin_path']),
+            realpath(Config::projectDir() . $this->config['bin_path']),
```
This will cause the configuration to support only relative paths. Maybe we could detect if it is relative or absolute and use the projectDIr method accordingly?

mcustiel commented:
Hi. @zwirek Thanks for taking you time to create a PR. I made a comment on the diff.

zwirek commented:
Hi @mcustiel.
Your suggestion is right. I've changed my code to support both, relative and absolute paths. It seems that realpath will do the job.

I have also created new branch from master and changed PR destination to master, as develop branch seems to be outdated.
